src/sentry/discover/models.py

```diff
@@ -57,3 +60,18 @@ def set_projects(self, project_ids):
                     for project_id in new_project_ids
                 ]
             )
+
+
+class KeyTransaction(Model):
+    __core__ = False
+
+    # max_length here is based on the maximum for transactions in relay
+    transaction = models.CharField(max_length=200)
+    project = FlexibleForeignKey("sentry.Project", db_constraint=False)
+    owner = FlexibleForeignKey("sentry.User", null=True, on_delete=models.SET_NULL)
+    organization = FlexibleForeignKey("sentry.Organization")
```

Comment:
I think this foreign key will require updates to be done to deletions code in sentry/deletions/defaults/organization.py

Reply:
Added to `sentry/deletions/defaults/project.py` too